docs/library-specification.md

```diff
@@ -0,0 +1,208 @@
+This is the specification for the 3rd party library format to be used with Arduino IDE 1.5.x onwards.
+
+* rev.1 has been implemented starting with IDE version 1.5.3 (now superseded by rev.2)
+* rev.2 will be implemented starting from version 1.5.6
+* rev.2.1 will be implemented starting from version 1.6.10
+* rev.2.2 will be implemented starting from version 1.8.10
+
+This new library format is intended to be used in tandem with the Arduino IDE"s **Library Manager**, available since version 1.6.2. The Library Manager allows users to automatically download and install libraries needed in their projects, with an easy to use graphic interface. It doesn't yet take care of dependencies between libraries.
```

Comment:
```suggestion
This new library format is intended to be used in tandem with the Arduino IDE's **Library Manager**, available since version 1.6.2. The Library Manager allows users to automatically download and install libraries needed in their projects, with an easy to use graphic interface. It doesn't yet take care of dependencies between libraries.
```

Reply:
I also noticed this, as well as some other issues, but I thought it would be best for this PR to be as much a straight across migration of the content as possible, since I know all the commits will be squashed. This typo exists in the original content, it's not something I introduced during the migration.

I had planned to submit a follow up PR to fix the issues I noticed while I was doing the migration.

That said, if you want me to fix this in this PR, I'm happy to do so. I'm glad I'm not the only one who notices and cares about these sorts of details.

Reply:
We can merge and fix it right after, I have no objections, so let's wait for @cmaglie review on the topic level then.